routes/index.js

```diff
@@ -1,17 +1,25 @@
 var express = require("express");
 var router = express.Router();
-const csrf = require("csurf");
-const csrfProtection = csrf({ cookie: true });
+const csrf = require('csurf');
+const csrfProtection = csrf({cookie: true});
+const { Sequelize } = require('sequelize');
+const Op = Sequelize.Op;
 
+const { User, Movie, Review, sequelize } = require('../db/models');
+const asyncHandler = handler => (req, res, next) => handler(req, res, next).catch(next);
 
 
-/* GET home page. */
-router.get("/", csrfProtection, function (req, res, next) {
-  res.render("index", {
-    token: req.csrfToken(),
-    title: "a/A Express Skeleton Home",
-  });
-});
+router.get('/', csrfProtection, asyncHandler(async(req, res, next) => {
+  const topMovies = await Movie.findAll({where: { voteRating:{ [Op.gt]: 8 }, voteCount: { [Op.gt]: 2000 } }, order: [['voteRating', 'DESC']], limit: 5});
+  const recentReviews = await Review.findAll({ include: [Movie, User], order: [['updatedAt', 'DESC']], limit: 5});
+  function randomNum(max) {
+    return Math.floor(Math.random() * Math.floor(max));
+  };
+  const randomBest = randomNum(880);
```

Comment:
Do you know that there will always be 880 movies in the database? If there are ever any fewer then the app would try to query a primary key that doesn't exist